lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -313,7 +313,7 @@ def translate_hash_to_trigger(puppet_trigger)
       }
 
       trigger['type']['days_of_week'] = if puppet_trigger['day_of_week']
-                                          PuppetX::PuppetLabs::ScheduledTask::Trigger::V1::Day.bitfield_from_days_of_week(puppet_trigger['day_of_week'])
+                                          PuppetX::PuppetLabs::ScheduledTask::Trigger::V1::Day.names_to_bitfield(puppet_trigger['day_of_week'])
```

Comment:
Much easier to follow with the renaming

Reply:
nit: Typically that's a bitmask not a bitfield.  I don't think it's worth changing though, particularly if it's used everywhere

Reply:
Yeah, the data structure is as close to a bitfield as Ruby can manage. But bitmask is certainly the more common term - I'll change it as long as there isn't too much carnage.